smssync/src/main/java/org/addhen/smssync/presentation/util/Utility.java

```diff
@@ -278,6 +278,9 @@ public static String getPhoneNumber(@NonNull Context context, @NonNull PrefsFact
      * @return String
      */
     public static String capitalizeFirstLetter(String text) {
+        if(text == null) return null;
```

Comment:
Heh. Just trying to maintain consistency. In fact it doesn't matter. Make the call.